test/hotspot/jtreg/compiler/lib/compile_framework/CompileFramework.java

```diff
@@ -0,0 +1,254 @@
+/*
+ * Copyright (c) 2024, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+package compiler.lib.compile_framework;
+
+import compiler.lib.compile_framework.SourceCode;
+import compiler.lib.compile_framework.CompileFrameworkException;
+import compiler.lib.compile_framework.InternalCompileFrameworkException;
```

Comment:
What do you mean? I have multiple uses in the file.

Reply:
I want to separate exceptions where the user is responsible, and internal exceptions that should never happen (otherwise there is a bug in the CompileFramework).

Reply:
Aren't they in the same package (and therefore the imports are redundant)?

Reply:
Correct, fixing that. Thanks!